_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,176 @@
+For now, we support:
+- [ParmEd objects](https://userguide.mdanalysis.org/stable/formats/reference/parmed)
+- [the chemfiles library](https://userguide.mdanalysis.org/stable/formats/reference/chemfiles)
+
+New additions to analysis include:
+- `frames` and `times` attributes to AnalysisBase to capture the frames and times that the analysis was `run()` on. This is accessible to all analyses that subclass AnalysisBase.
+- a [correlations module](https://docs.mdanalysis.org/stable/documentation_pages/lib/correlations.html) for computing the discrete autocorrelation function
+- a [HydrogenBondAnalysis](https://docs.mdanalysis.org/stable/documentation_pages/analysis/hydrogenbonds.html) class for improved and more efficient analysis of hydrogen bonds
+- an [AverageStructure](https://docs.mdanalysis.org/stable/documentation_pages/analysis/align.html#MDAnalysis.analysis.align.AverageStructure) class for computing the average structure of a trajectory out of memory
+- a [hole2](https://docs.mdanalysis.org/stable/documentation_pages/analysis/hole2.html) module for improved interfacing with the HOLE2 program
+- a [DensityAnalysis](https://docs.mdanalysis.org/stable/documentation_pages/analysis/density.html#MDAnalysis.analysis.density.DensityAnalysis) class for improved density analysis
+- a method to compute the root-mean-square-inner-product of subspaces
+- a method to calculate the cumulative overlap of a vector in a subspace
+
+Other additions to core functionality include:
+- [wrap and unwrap on-the-fly transformations](https://docs.mdanalysis.org/stable/documentation_pages/transformations/wrap.html)
+- methods to add [bonds][add_bonds], [angles][add_angles], [dihedrals][add_dihedrals], and [impropers][add_impropers] to a Universe
+- methods to delete [bonds][del_bonds], [angles][del_angles], [dihedrals][del_dihedrals], and [impropers][del_impropers] from a Universe
+- the [get_connections][] method to get bonds, angles, dihedrals and impropers in an AtomGroup, with the ability to specify whether all atoms involved is within the AtomGroup (`outside=False`), or whether *at least one* atom involved is within the AtomGroup (`outside=True`)
+
+Miscellaneous performance improvements include:
+- Dihedral selection in the Ramachandran class has been sped up ~700x.
+- TPR parsing has been sped up 2–30x.
+
+## Notable improvements
+
+We have added significant flexibility to our atom selection language.
+
+Notable changes to analysis include:
+- The argument order to AnalysisFromFunction are now as specified in the documentation
+- The `select` keyword has been standardized by removing `selection`, `atomselection`, and `ref_select` in the `contact`, `gnm`, `helanal`, `hole`, `encore`, and `hydrogenbonds` modules
+- The `save()` functions have been removed from `contacts`, `diffusionmap`, `hole`, `lineardensity`, and `rms` modules
+- `PCA(align=True)` now correctly aligns the trajectory
```

Comment:
Should we link to the issue here? This is the type of thing I would expect folks to want to know how it affected their analyses.

Reply:
good idea

Reply:
On that note: link to issues for neighbor searching? After all, NF paid @richardjgowers to fix them (small dev grant).

Reply:
There might be a more elegant way to link the issues, but I've added something for now